### PR TITLE
emscripten 2.0.0

### PR DESCRIPTION
**Pull request by zachauten**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Review by SMillerDev — DISMISSED** [review]

**Comment by SMillerDev** [comment]
@zachauten could you update the resources?

File: Formula/emscripten.rb [hunk 1]
```diff
@@ -8,18 +8,8 @@ class Emscripten < Formula
...
-
-    resource "fastcomp" do
-      url "https://github.com/emscripten-core/emscripten-fastcomp/archive/1.40.1.tar.gz"
-      sha256 "c34868ab566e9f073df319d9872608cef47ed1ea74852acacb12a22fd7c99a4c"
-    end
-
-    resource "fastcomp-clang" do
-      url "https://github.com/emscripten-core/emscripten-fastcomp-clang/archive/1.40.1.tar.gz"
-      sha256 "9ce4612df39684348d78acb711ec10bee98ad4ac136fb0dcb70d4c884b8bb6b3"
-    end
```
Comment (sbc100) [inline Formula/emscripten.rb:22]: You probably want to replace these with https://github.com/llvm/llvm-project.

Comment (zachauten) [reply Formula/emscripten.rb:22]: Thanks! I misread the other thread.

**Comment by zachauten** [comment]
I think this needs to wait until emscripten can work with brew's llvm.

**Comment by sbc100** [comment]
Why can't we just build llvm in the same way we built fastcomp previously?

While it would be nice if emscripten one day supports LLVM stable that plan has not be finalized.

**Comment by MaxGraey** [comment]
> I think this needs to wait until emscripten can work with brew's llvm.

I guess it never happening

**Pull request closed without merging**